### DISCUSSION TITLE
`CMD + H` not hiding app in Markdown Pro Editor

### DISCUSSION THREAD

**Discussion by martynchamberlin** [standardnotes/forum#2476] [General] [answered]
**Describe the bug**
Instead of hiding Standard Notes when typing `CMD + H`, in the Markdown Pro Editor it causes the current line to turn into an H1 headline. 

**To Reproduce**
1. Create a new note with the Markdown Pro theme.
2. Type `CMD + H`
3. `# ` is inserted on the currently selected line. 

**Expected behavior**
Rather than `#3` above, instead app should disappear. 

**Desktop**
 - OS: macOS Big Sur 11.5.2

**Comment by JaspalSuri** [comment]
Hi @martynchamberlin, sorry to hear that. Since this feature might be used by several users, we might not change this unless we receive more reports about this. I would suggest using `CMD+M` to minimize the app in the meantime.

**Comment by gilletty** [comment]
I'm also using macOS and agree with @martynchamberlin. Perhaps in the case of system conflicts, an additional modifier key should be used, such as `CMD-SHIFT-H`.

**Comment by JaspalSuri** [comment]
Hi @gilletty, thanks for the suggestion. I'll pass this along to the dev team for further consideration. 🙂

**Comment by myreli** [comment, marked as answer]
Unfortunately, due to the secure plugin architecture, 3rd party editors can't access the system level as they are rendered inside an iframe. Thus it's unable to perform actions outside of the editor panel. 

However, while this can not be done for the markdown editor, you can `CMD + H` using one of the in-house editors, Plain or Super or anywhere else within the app. Note that, for Super, you need to `esc` to leave the editor context. 

I hope this helps. Let us know if you run into any issues!
